### ISSUE TITLE
[Request]  python-jaxlib-cuda &  python-jaxlib

### ISSUE THREAD

**Issue by 3ri4nG0ld** [chaotic-aur/packages#2966]
### Package:

https://aur.archlinux.org/packages/python-jaxlib-cuda
https://aur.archlinux.org/packages/python-jaxlib

### Purpose:

is a library

### Copyright:

Apache-2.0 license

### Expected Interest:

Some

### Already available?

No

### Unique request?

No

### Banned package?

No

### More information:

They are very slow to compile, they would help a lot.

**Comment by xiota** [comment]
What package, program, or purpose does this library serve?  Without additional info, this package qualifies as a dependency without dependent.

**Comment by 3ri4nG0ld** [comment]
python-jax
https://aur.archlinux.org/packages/python-jax

**Comment by xiota** [comment]
`python-jax` has the same issue – dependency without dependent.  Same for any cluster of `python-*` packages.  Describe your use case.

**Comment by 3ri4nG0ld** [comment]
I really only have this package installed to use sdiff-gtk-git, which requires python-transformers which in turn requires  python-flax which in turn reequires python-jax

I understand that there are really a lot of intermediate packages for a single program, so I suppose that having alternatives as is the case it does not matter much.

**Comment by xiota** [comment]
That is the info I wanted to know.  Please edit the initial request to add it.

Is there some reason to request only one of the libraries instead of the program you really want to use?

**Comment by 3ri4nG0ld** [comment]
Not really, I only put the libraries because they were the ones that were updated more frequently and took longer to compile.

**Comment by Technetium1** [comment]
@jzbor can you please update https://aur.archlinux.org/packages/sdiff-gtk, we would like to provide it :)

**Comment by jzbor** [comment]
Hi, I have had some problems maintaining it over the past, as some of the libraries used had some issues. I am no longer using Arch (and not even sdiff-gtk to be honest), but I can try to update the PKGBUILD in the holidays. Feel free to ping me again if I forget about it :grimacing: .

Also it should be noted that sdiff-gtk has some very rough edges, e.g. not showing any sort of download progress. I hacked it together in a weekend being jealous of my friends cool AI-apps that sadly ran on Mac only. But I am glad that you seem to enjoy it and will try to provide a working PKGBUILD.

**Comment by xiota** [comment]
For this package, to simplify the depends tree, should bypass check on `aur/python-safetensors` and skip optdeps for `aur/python-transformers`.

@jzbor Please add me as comaintainer.

**Comment by jzbor** [comment]
I recently tried packaging the project for NixOS and failed. Python is sort of my arch nemesis when it comes to packaging and I currently do not have the time to fix or update the AUR package, but I have added @xiota as co-maintainer. Feel free to update and improve the package as you see fit.

In general I want to use this opportunity to make clear the support status of `sdiff-gtk`: I don't really actively use it myself as I don't own a powerful enough graphics card for it to run in an acceptable speed.  Therefore I will most probably not provide any feature updates or substantial improvements, however I will try to update the dependencies to mitigate any Security Alerts when they pop up.

I hope you have fun using `sdiff-gtk` and wish you the best of luck packaging it :)
